targets/gnome

```diff
@@ -8,9 +8,15 @@ HOSTBIN='startgnome'
 CHROOTBIN='crouton-noroot startgnome gnome-session-wrapper'
 . "${TARGETSDIR:="$PWD"}/common"
 
+if release -ge wheezy -ge kali -ge trusty; then
```

Comment:
I'm assuming all modern distros will be using -flashback, correct?  In that case, it might be better to make the if statement check `release -lt trusty`, since then wheezy+, kali+, and anything else we add will work without modification.